api/groups/vmValuesGroup_test.go

```diff
@@ -480,11 +465,11 @@ func TestVMValuesGroup_UpdateFacade(t *testing.T) {
 		require.Equal(t, hex.EncodeToString(valueBuff), response.Data)
 
 		newFacade := &mock.FacadeStub{
-			ExecuteSCQueryHandler: func(query *process.SCQuery) (vmOutput *vm.VMOutputApi, e error) {
+			ExecuteSCQueryHandler: func(query *process.SCQuery) (*vm.VMOutputApi, api.BlockInfo, error) {
 
 				return &vm.VMOutputApi{
 					ReturnData: nil,
-				}, expectedErr
+				}, api.BlockInfo{}, expectedErr
```

Comment:
please add a test (or update an existing one) where the blockinfo isn't empty

Reply:
added